.circleci/config.yml

```diff
@@ -65,6 +73,10 @@ jobs:
           command: |
               npm run test-server:ci $(circleci tests glob "server/**/test/*.js" "server/**/test/*.jsx" | circleci tests split --split-by=timings)
 
+      - run:
+          name: Upload lockfile changes
+          command: greenkeeper-lockfile-update
```

Comment:
@blowery I think this should be `greenkeeper-lockfile-upload`, but I don't know how to actually test this 😅

Reply:
nice catch! i'm going to try out renovate instead for a bit, as it offers the same lockfile updating but also lets us bundle monorepo updates together (for babel / webpack / etc)... if we end up coming back to greenkeeper, i'll definitely fix this.